### ISSUE TITLE
Scan OIG reports for graphical elements and recommendations

### ISSUE THREAD

**Issue by JonellaCulmer** [fecgov/fec-cms#3787]
**What we're after:**
OIG recommended the website team take a look at OIG reports to get a better idea of the function and needs for their office

Consider:
- Graphs and images
- Are there things we can highlight that would benefit the regulated community or individuals who file OIG complaints?
- Scan inventory already on OIG page to figure out what is useful and what is not 
- Picture of IG

### Related issues
- #3901 

### Completion criteria
- [x] Document key findings from report that support improved user experience for OIG
- [ ] Compile recommendations for future state/landing page

**Future steps:**
- Jonella and Dorothy: Scan some of the reports to see what OIG does
- Take a look at the hotline portal

**Comment by JonellaCulmer** [comment]
One recommended link to review: https://www.fec.gov/resources/cms-content/documents/OIG_review_of_FEC_outstanding_recommendations_as_of_February_2020.pdf

**Comment by JonellaCulmer** [comment]
Here's a collection of some charts and other visualizations featured in previous OIG reports. I have to acknowledge that these visualizations are entirely out of context, so don't read anything into them without seeking out the full reports. 

<img width="336" alt="Screen Shot 2020-07-22 at 1 11 45 PM" src="https://user-images.githubusercontent.com/31663028/88206948-149a5d00-cc1d-11ea-8f57-0012cfc14bfe.png">
<img width="681" alt="Screen Shot 2020-07-22 at 11 57 07 AM" src="https://user-images.githubusercontent.com/31663028/88206954-1532f380-cc1d-11ea-942a-a1c8bb0e503b.png">
<img width="645" alt="Screen Shot 2020-07-22 at 11 57 17 AM" src="https://user-images.githubusercontent.com/31663028/88206955-1532f380-cc1d-11ea-95d1-f715f679986b.png">
<img width="645" alt="Screen Shot 2020-07-22 at 11 57 25 AM" src="https://user-images.githubusercontent.com/31663028/88206956-15cb8a00-cc1d-11ea-912e-da0efb590bc9.png">
<img width="646" alt="Screen Shot 2020-07-22 at 12 08 29 PM" src="https://user-images.githubusercontent.com/31663028/88206957-15cb8a00-cc1d-11ea-8a28-78312ba51029.png">
<img width="529" alt="Screen Shot 2020-07-22 at 12 08 46 PM" src="https://user-images.githubusercontent.com/31663028/88206958-15cb8a00-cc1d-11ea-8c88-1a38f9bfac71.png">

**Comment by JonellaCulmer** [comment]
Here is a collection of charts and other visualizations featured on other OIG websites. Again, please don't read anything into these charts, they are out of context and are from multiple agencies.

<img width="464" alt="Screen Shot 2020-07-22 at 1 16 31 PM" src="https://user-images.githubusercontent.com/31663028/88207870-68f20c80-cc1e-11ea-956e-8db28b73bd82.png">
<img width="479" alt="Screen Shot 2020-07-22 at 1 16 53 PM" src="https://user-images.githubusercontent.com/31663028/88207874-68f20c80-cc1e-11ea-9865-da9f45f36e4d.png">
<img width="474" alt="Screen Shot 2020-07-22 at 1 17 00 PM" src="https://user-images.githubusercontent.com/31663028/88207876-68f20c80-cc1e-11ea-9988-10ea2e0d1b77.png">
<img width="172" alt="Screen Shot 2020-07-22 at 1 17 15 PM" src="https://user-images.githubusercontent.com/31663028/88207877-68f20c80-cc1e-11ea-8074-80811d1f863e.png">
<img width="411" alt="Screen Shot 2020-07-22 at 1 17 29 PM" src="https://user-images.githubusercontent.com/31663028/88207878-698aa300-cc1e-11ea-91e9-a8f15953763b.png">

**Comment by JonellaCulmer** [comment]
Using the graphics available from recent/previous OIG reports as a guide and comparing them to the graphics showcased by other agency OIG, our OIG may have a path forward to showcasing their own. 

At present, the current visualizations could provide a stepping stone to new visualizations. These would likely be based on the very high-level graphics from reports - but this would be dependent on the visualizations we know to be helpful to our users. I refer specifically to this type of visualization: 
<img width="645" alt="Screen Shot 2020-07-22 at 11 57 25 AM" src="https://user-images.githubusercontent.com/31663028/88206956-15cb8a00-cc1d-11ea-912e-da0efb590bc9.png">

My next steps would be to do some additional research about the expectations of OIG visitors and get a better understanding of their needs. Whether they include other OIG team representatives, individuals interested in agency or FEC oversight, or individuals who engage with and submit complaints.

**Comment by JonellaCulmer** [comment]
I would like to reach out to other OIG offices to see if they've done any user research to guide their visualization decisions. I'll contact our OIG first to get their feedback first before pursuing this track. 

Moving to blocked while I wait to hear back from the OIG.

**Comment by JonellaCulmer** [comment]
Interview questions for DOL OIG: 

- When building your pages, what were your main goals?
- The website for the OIG has a different appearance than other DOL pages. Can you explain why you made that choice?
- Did you have and use any customer feedback when determining which content to show?
- What were your goals when identifying the high level visualizations for your pages?
- How did you determine which visualizations to showcase?
- How often do you update your visualizations?
- Do you make updates to your pages independently, or do you have website support to make updates or changes?
- Anything else that you wanted to share about your website or data visualizations?

**Comment by JonellaCulmer** [comment]
Before breaking down the individual elements that are needed for an OIG landing page, I want to summarize what we know about the OIG’s and it’s customer’s needs: 

- Consider breaking up content into separate content-specific pages with in-page navigation
- High-level data visualizations that are easy to maintain
- Quick access to the report search
- Showcase recent reports
- Quick access for file complaints and understand what can and can't be reported to the OIG

Feedback from DOL OIG:
- Most of their visitors are looking at how to file a complaint.
- Gear their page towards their biggest stakeholders, who for them are press and Congress
- Like using images and graphics/infographics as much as possible to convey the information without requiring visitors to read the reports. 
- In selecting images, the fall back on balancing what's important  with what they have to report.
- For their dashboard, it's based on what's required of them to report from their semi-annual report and the IG act. Use their agency bar chart as that reporting mechanism as well as an incentive to getting recommendations addressed.
- Their dashboard is automated, using a spreadsheet, and only requires spot checks, normally once a week from an OIG team member to verify the information.

**Comment by JonellaCulmer** [comment]
Based on the visualizations our OIG currently includes in their tickets, using the wisdom of the DOL OIG to build visualizations based on data sets that already exist, and the fact that we've recently built a couple tables based on spreadsheets, I think we're in a good position to create easy to maintain charts or graphics for the OIG. Some additional conversations need to happen, but I think we can recommend at this stage that we see what data sets the OIG maintains and identify which ones they'd/we'd like to showcase on the OIG landing page. 

More to come on this in the next ticket for the landing page mock-up. We can incorporate what we know so far and discuss with them in a followup discussion about which datasets they'd like to add to the OIG landing page. 

Closing this ticket in favor of #3946.
